cmd/oras/root/repo/tags.go

```diff
@@ -70,6 +72,9 @@ Example - [Experimental] Show tags of the target repository in JSON view:
 
 Example - [Experimental] Show tags of the target repository with Go template:
   oras repo tags localhost:5000/hello --format go-template --template "{{.tags}}"
+
+Example - [Experimental] Show tags of a specific repository in OCI layout:
+  oras repo tags --oci-layout-path layout-dir localhost:5000/hello
```

Comment:
It seems like a `--filter` option would have made more sense. I really hadn't thought about it before, but this is an odd syntax with `--oci-layout` and `--oci-layout-path`. Not for this PR though.